src/libstore/globals.hh

```diff
@@ -629,6 +629,11 @@ public:
 
           Files fetched via `NIX_PATH`, `fetchGit`, `fetchMercurial`,
           `fetchTarball`, and `fetchurl` respect this TTL.
+
+          This option also affects how often the global flake registry is
+          updated.  For example, a flake URI like `nixpkgs#hello` will trigger
+          a new download of Nixpkgs if the last download is older than the
+          `tarball-ttl`.
```

Comment:
```suggestion
          This option also affects how often the global flake registry is
          updated, and how often individual flake inputs are updated.
          For example, a flake URI like `nixpkgs#hello` will trigger
          a new download of Nixpkgs if the last download is older than the
          `tarball-ttl`.
```

I don't think we need an example of the flake registry updating, because that thing doesn't update very often anyway. Analogy with DNS holds up.

OT: Should we add the analogy with DNS to the flake registry docs?